.github/workflows/swift.yml

```diff
@@ -0,0 +1,18 @@
+name: Swift
+
+on:
+  push:
+    branches: [ staging ]
```

Comment:
Probably this should be changed to master since we don't have any complex staging process yet 🤔

Reply:
This doesn't necessarily have to map to anything for now in the main repo -- however, for the sake of others who might fork the app to work on it, having a staging branch that they can simply push to and get CI actions going before making a pull request would make writing a development guide much simpler.

That, and a staging branch could be added in the future, and it would only apply at that time. For those reasons, I argue that it should stay as is.